src/AbpDevTools/Commands/DatabaseDropCommand.cs

```diff
@@ -0,0 +1,81 @@
+using System.Diagnostics;
+using AbpDevTools.Notifications;
+using CliFx.Infrastructure;
+
+namespace AbpDevTools.Commands;
+
+[Command("database-drop", Description = "Drops all databases in the working directory")]
+public class DatabaseDropCommand : ICommand
+{
+    [CommandParameter(0, IsRequired = false, Description = "Working directory to search for EntityFramework projects. Default: . (Current Directory)")]
+    public string WorkingDirectory { get; set; }
+    
+    [CommandOption("force", 'f')]
+    public bool Force { get; set; }
+    
+    protected readonly INotificationManager notificationManager;
+
+    public DatabaseDropCommand(INotificationManager notificationManager)
+    {
+        this.notificationManager = notificationManager;
+    }
+
+    public async ValueTask ExecuteAsync(IConsole console)
+    {
+        if (string.IsNullOrEmpty(WorkingDirectory))
+        {
+            WorkingDirectory = Directory.GetCurrentDirectory();
+        }
+
+        var efCoreProjects = Directory.EnumerateFiles(WorkingDirectory, "*.csproj", SearchOption.AllDirectories)
+            .Where(x => x.EndsWith("EntityFrameworkCore.csproj"))
+            .Select(x => new FileInfo(x))
+            .ToList();
```

Comment:
We can show a spinner while searching files, when the command is executed in a very root folder like `D:\`, the process can take some times and user shouldn't think it's stuck

Here is an example:
https://github.com/enisn/AbpDevTools/blob/f22b39401acca0899108aff2c7346b9a598f69f3/src/AbpDevTools/Commands/BuildCommand.cs#L122-L140

Reply:
![abpdevtools-spinner](https://github.com/enisn/AbpDevTools/assets/23705418/fa11574a-6598-4838-a595-d2cb3fe77de0)

Reply:
Sure, I'll add spinner 👍